lib/membrane/rtcp/receiver.ex

```diff
@@ -20,12 +20,13 @@ defmodule Membrane.RTCP.Receiver do
               remote_ssrc: [spec: RTP.ssrc_t()],
               report_interval: [spec: Membrane.Time.t() | nil, default: nil],
               fir_interval: [spec: Membrane.Time.t() | nil, default: nil],
-              telemetry_metadata: [spec: [{atom(), any()}], default: []]
+              telemetry_label: [spec: Membrane.TelemetryMetrics.label(), default: []]
 
-  @event_name [:sending_fir, :rtcp]
+  @event_name [:RTCP, :fir, :sending]
```

Comment:
Wouldn't it be better to name this `sent` instead of `sending`?

```suggestion
  @event_name [:RTCP, :fir, :sent]
```